packages/components/src/components/DiagramSequence.vue

```diff
@@ -111,6 +132,9 @@ export default {
   },
 
   methods: {
+    loadData(data) {
+      this.serializedDiagram = data;
```

Comment:
If this is the primary means of loading a diagram, `serializedDiagram` should be a `data` element instead of a `prop`. `prop` data is meant to be controlled by a parent component only.

Reply:
serializedDiagram can also be set directly, to render the diagram without the AppMap wrapping stuff.